plugins/generator-1.15.2/forge-1.15.2/templates/fluid.java.ftl

```diff
@@ -187,6 +204,53 @@ import net.minecraft.block.material.Material;
 		</#if>
 	}
 
+	public static abstract class CustomFlowingFluid extends ForgeFlowingFluid {

```

Comment:
extending ForgeFlowingFluid is only needed if spawnParticles is enabled, so consider making whole this class only generate when this is enabled to keep code cleaner in case spawnParticles is not used